xcconfigs/Project.xcconfig

```diff
@@ -61,6 +61,9 @@ SWIFT_VERSION                      = 5.0
 VERSION_INFO_PREFIX                = 
 VERSIONING_SYSTEM                  = apple-generic
 
+// Catalyst
+SDKROOT                            = iphoneos
```

Comment:
I don't have info on this and will this be OK with the regular mac?

Reply:
I have tested by creating a release package and used `.framework` inside a sample mac app.